skymp5-client/src/front/skympClient.ts

```diff
@@ -133,7 +133,7 @@ export class SkympClient {
 
     on("update", () => {
       if (!this.singlePlayer) {
-        this.sendInputs();
+        this.updateData();
```

Comment:
Please keep the original name. `updateData` is too common and doesn't say anything.